scripts/metadata_comparison/metadata_comparison/digester.py

```diff
@@ -0,0 +1,108 @@
+import argparse
+import json
+from metadata_comparison.lib import logging, operation_ids
+from metadata_comparison.lib.operation_ids import CallNameSequence, JsonObject, OperationId
+from metadata_comparison.lib.comparison_paths import ComparisonPath
+
+import dateutil.parser
+from typing import AnyStr, Dict
+
+Version = "0.0.1"
+verbose = False
```

Comment:
These two possibly belong more in the "if main" method? Wouldn't want to inadvertently import them in other scripts

Reply:
Version is being used in two functions, verbose in zero. 🤔 

I think the lesson here may be not to `import *`. 😉